src/hooks/use-import-export.tsx

```diff
@@ -101,9 +101,16 @@ export const ImportExportProvider = ( { children }: { children: React.ReactNode
 			try {
 				await stopServer( selectedSite.id );
 
+				let filePath: string;
+				if ( file instanceof File ) {
+					filePath = getIpcApi().getPathForFile( file );
+				} else {
+					filePath = file.path;
+				}
```

Comment:
```suggestion
				const filePath = file instanceof File ? getIpcApi().getPathForFile( file ) : file.path;
```
We could make this slightly more compact